creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -126,6 +139,7 @@ const updateReplicaSetJobProcessor = async function ({
     logger.error(
       `ERROR issuing update replica set op: userId=${userId} wallet=${wallet} old replica set=[${primary},${secondary1},${secondary2}] | Error: ${e.toString()}`
     )
+    logger.error(e.stack)
```

Comment:
can we just change above call to e.stack to have single log?

Reply:
same for above

Reply:
changed